pyam/utils.py

```diff
@@ -146,25 +146,34 @@ def read_files(fnames, *args, **kwargs):
     for fname in fnames:
         logger().info('Reading `{}`'.format(fname))
         df = read_pandas(fname, *args, **kwargs)
-        dfs.append(format_data(df))
+        df = format_data(df)
+        dfs.append(df)
 
     return pd.concat(dfs)
 
 
 def format_data(df):
     """Convert an imported dataframe and check all required columns"""
+    # all lower case
+    df.rename(columns={c: str(c).lower() for c in df.columns}, inplace=True)
+
+    if 'notes' in df.columns:  # this came from the database
+        logger().info('Ignoring notes column in dataframe')
+        df.drop(columns='notes', inplace=True)
+        col = df.columns[0]  # first column has database notes
```

Comment:
please rephrase to 'copyright notice' or similar, the word 'notes' is confusing with the previous line